packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -69,13 +72,44 @@ export class AliasGatherer implements ParametersGatherer<Alias> {
     if (alias === undefined) {
       return { type: 'CANCEL' };
     }
-    return alias === ''
-      ? { type: 'CONTINUE', data: { alias: defaultAlias } }
-      : { type: 'CONTINUE', data: { alias } };
+    const expirationDays = {
+      prompt: nls.localize(
+        'parameter_gatherer_enter_scratch_org_expiration_days'
+      ),
+      value: defaultExpirationdate
```

Comment:
For the expiration days, which has an actual default, use **value** in the InputBoxOptions instead of **placeHolder**. The difference here is that hitting enter on the placeholder returns an undefined and since we're defaulting to 7 there's no need to force the user to do additional typing, they can just hit enter to accept the default value. The InputBox does highlight the default value so if the user opts to change it they don't have to delete it.